### ISSUE TITLE
Search is not working

### ISSUE THREAD

**Issue by alfish2000** [owncloud/tasks#117]
Using the Task Search (in left panel) only shows an empty main window.
I see the URL changing to e.g.  ".../apps/tasks/#/search/test" when I search for the word test.
But the main window never shows any result.
Using the ownCloud search (upper right corner) only shows other hits (e.g. test file, test calendar entry, ...) but not the test task in Tasks App.

Tested with Tasks v0.5 and the latest sources from GitHub.
I'm using ownCloud 7.0.4 (stable) on Ubuntu 14.04.
Installed ownCloud via apt-get (with previously added repository: http://download.opensuse.org/repositories/isv:/ownCloud:/community/xUbuntu_14.04 etc)

**Comment by raimund-schluessler** [comment]
Thanks for the report, you are right, the app intern search seems to be broken. I will need to have a look why this is.

The ownCloud search is not implemented for the tasks app, so it won't show anything related to tasks at all. This could be enhanced, but might be some work.

**Comment by raimund-schluessler** [comment]
The in-app search should work again now. I opened a separate issue #120 to implement the global owncloud search.

**Comment by alfish2000** [comment]
Thanks so much for the quick fix.
I've tested it quickly and the search works fine now.
Tested in task title, task description and comments.

I had a look in the source code before too but I had no idea where to fix it.
I've just stepped a little through the code with firefox javascript debugger :-)

So if you have any other fixes to test I would be glad to help.
I've installed ownCloud and Tasks a week ago for the first time and I'm very interested to replace my previous task list (just an Android APP) with this ownCloud server/client approach (I'm using Mirakel at the moment to sync via DavDroid to ownCloud and I especially like the web based Tasks in ownCloud too)
